.eslintignore

```diff
@@ -0,0 +1,2 @@
+packages/*/es/
+packages/*/lib/
```

Comment:
A wildcard pattern could break eslint check if we create new sub directory named `lib` or `es` *(Sometimes we may use mono package for internal use, and not output as a npm package)*. I think it's better to use build dir prefix with wildcard pattern. (maybe `packages/*/build/es`?)

Reply:
I only applied one `*` as the ignore pattern instead of the double-asterisk `**`. It should be all good since the lint rule will just ignore a file like `packages/ui/es/index.js` but not `packages/ui/src/lib/index.js`.

Hence if we'll be following the same convention that assuming 'es' and 'lib' folders as the generated folders for the mono repos, it would address the potential issue which you concerned about.

Besides, if looking at the [.fatherrc](https://github.com/pingcap/tug-website/blob/CPT-89-publish-ui-package/packages/ui/.fatherrc.js), you may find the `es` and `lib` folders conventions are the default/suggested options provided by [father-build](https://github.com/umijs/father). We'd better keep what it is before any issue being encountered.

BTW, for our internal consumption, [I did a slight update to map `'@tidb-community/ui'` to its source folder](https://github.com/pingcap/tug-website/pull/38/commits/ad0ce028ae5c4cd0599e49c233fc98092480e391). I think it will give us an instant building result if launching with the local dev server.